FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+ErrorHandling.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ComposeViewController+ErrorHandling.swift
+//  FlowCrypt
+//
+//  Created by Roma Sosnovsky on 14/09/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - Error handling
+extension ComposeViewController {
+    func requestMissingPassPhraseWithModal(for signingKey: Keypair, isDraft: Bool = false, withDiscard: Bool = false) {
```

Comment:
On Android, I think we went with not signing drafts, and that way we could avoid this. But I think either way is fine. Highlighting the difference.